cds/Module.md

```diff
@@ -0,0 +1,111 @@
+# CDS Base Module
+
+## Sample Usage
+
+This section focuses on samples depicting how to use this package to implement CDS related integrations
+
+### Prerequisites
+
+1. Install Ballerina 2201.6.0 or later
```

Comment:
Let's use bal 8.2 or later

Reply:
Add link to the ballerina installation page

Reply:
Done

Reply:
https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/372/commits/2e26b5e51cc31bfb73f40f08c19ae6e648926992